AGENTS.md

```diff
@@ -0,0 +1,53 @@
+### Project Summary: Lightning Loop
+
+**1. Core Purpose:**
+Lightning Loop is a non-custodial service from Lightning Labs for bridging the Bitcoin mainnet (on-chain) and the Lightning Network (off-chain). It enables users to rebalance their channels—moving funds into or out of Lightning—without the need for closing and reopening them. The system is architected to be non-custodial, using atomic swaps to ensure neither the user nor the server can lose funds.
```

Comment:
...their channels by moving funds into or out of Lightning - without the need for closing and...

Reply:
Fixed